HCL.IdentityServer.API.BLL/Services/RegistrationService.cs

```diff
@@ -26,22 +26,23 @@ public RegistrationService(ILogger<RegistrationService> logger, IOptions<JWTSett
             _options = options.Value;
             _accountService = accountService;
         }
-        public async Task<BaseResponse<(string, Guid)>> Registration(AccountDTO DTO)
+        public async Task<BaseResponse<AuthDTO>> Registration(AccountDTO DTO)
```

Comment:
Do you have global excrption handler?

Reply:
I don't use global exception handler in identity server